versioneer.py

```diff
@@ -0,0 +1,2109 @@
+
```

Comment:
```suggestion
#!/usr/bin/env python
```
Also would be nice if this file was made user-executable to git.

Reply:
I'm actually unsure why versioneer added this file. Maybe it does something if you have a setup.py, but for us, it isn't actually used. The important bit is the code in _version.py that parses the version from the git tag.